ARM-Samples/Create-Account/README.md

```diff
@@ -17,7 +17,7 @@ The resource will be deployed to your subscription and will create the Azure Vid
 
 ## Prerequisites
 
-* An Azure Video Analyzer for Media (AMS) account. You can get one for free through the [Create AMS Account](https://docs.microsoft.com/en-us/azure/media-services/latest/account-create-how-to).
+* An Azure Media Services (AMS) account. You can get one for free through the [Create AMS Account](https://docs.microsoft.com/en-us/azure/media-services/latest/account-create-how-to).

```

Comment:
Not a fan of this sentence "You can get one for free through the"
Maybe "You can create a free trial account through the"